src/toastify.css

```diff
@@ -65,6 +65,7 @@
     left: 0;
     right: 0;
     max-width: fit-content;
+    max-width: -moz-max-content;;
```

Comment:
Shouldn't it be `-moz-fit-content`?
Also there is a double `;` in case you missed it :)